pyramid_swagger/api.py

```diff
@@ -4,26 +4,41 @@
 """
 import simplejson
 
+from pyramid_swagger.model import PyramidEndpoint
 
-def register_api_doc_endpoints(config):
-    """Create and register pyramid endpoints for /api-docs*."""
-    swagger_schema = config.registry.settings['pyramid_swagger.schema']
-    register_resource_listing(config, swagger_schema.resource_listing)
 
-    for name, filepath in swagger_schema.api_declarations.items():
-        with open(filepath) as input_file:
-            register_api_declaration(
-                config,
-                name,
-                simplejson.load(input_file)
-            )
+# TODO: document that this is now a public interface
```

Comment:
"this" is the fact that we're serving data at /api-docs? Or that it can be customized?